libsquoosh/src/codecs.ts

```diff
@@ -1,5 +1,28 @@
 import { promises as fsp } from 'fs';
-import { instantiateEmscriptenWasm, pathify } from './emscripten-utils.js';
+import { instantiateEmscriptenWasm, pathify } from './emscripten-utils';
```

Comment:
```suggestion
import { instantiateEmscriptenWasm, pathify } from './emscripten-utils.js';
```

I think we should keep file extensions around (TypeScript handles `.js` extensions correctly by default)